### DISCUSSION TITLE
Badge Icon IOS React Native on Push notification firebase

### DISCUSSION THREAD

**Discussion by CedricNavet** [community/community#128456] [Programming Help]
### Select Topic Area

Question

### Body

Hello everyone,

I have a native React application. I use Firebase on my server side to send push notifications. It works without any problem, I get my notification on my application taking into account the state of my app (foreground, background, quit). On Android, I manage to get a +1 icon on my application icon, which is called the Icon badge. But on IOS this is impossible.

I've tried to implement the explanations provided in the documentation https://rnfirebase.io/messaging/usage. The BackgroundMessageHandler event never fires on ios, which prevents me from adding a +1 icon to my app, regardless of its state, when I receive a notification. I've also tried using another package from the community but nothing works. https://github.com/react-native-push-notification/ios

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @CedricNavet ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
